src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUM_ITERATION = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < NUM_ITERATION; i++) {
+            Lottery lottery = new Lottery();
```

Comment:
Let's not create a new lottery for each iteration. I'd recommend creating Lottery before the loop